src/game.h

```diff
@@ -540,6 +560,8 @@ class Game
 
 		std::map<uint32_t, BedItem*> bedSleepersMap;
 
+		std::map<Tile*, uint8_t> cleanTiles;
```

Comment:
Looks like the value is unused, so this should be `std::set`

Reply:
was not aware of `std::set` thanks, will change it.